modules/ghservice/templates/apache/app_admin.conf.erb

```diff
@@ -0,0 +1,95 @@
+    ##
```

Comment:
The only two changes to the Apache conf that I added were for robots.txt and auth-protection - do you really want these in grasshopper-ui?